docs/guides/deploying_to_digital_ocean.md

```diff
@@ -0,0 +1,37 @@
+# Deploying with digital ocean
+
+There are multiple ways to deploy to digital ocean. We will deploy with docker swarm in this guide for simplicity.
+(note: while it's possible to use docker-compose it's generally not recommended for production. Since it's not much more difficult to use docker swarm, that's what we recommend.)
+
+## single machine with docker
+
+### deploying
+
+Prerequisite:
+
+- make sure you have completed the docker deployment guide
+- you know how to create a 'one-click app' droplet
+- you know how to ssh in a digital ocean droplet
+- you have published your image to dockerhub
+
+- Start by creating a new droplet. Click on the 'one-click apps' and choose docker. It will create a droplet for you with docker already installed
```

Comment:
No need to make each of the instructions here a list item, they should be paragraphs.